### PR TITLE
refs #15834 - rebuild bundler on Ubuntu/trusty

### PR DESCRIPTION
**Pull request by mmoll**
The Ubuntu package seems to be built for Ruby 1.8  and 1.9, while 1.9 and 2.0 is needed.

### CODE REVIEW

File: dependencies/trusty/bundler/bundler.dirs [hunk 1]
```diff
@@ -0,0 +1,2 @@
+var/lib/gems/1.9.1
+var/lib/gems/2.0.0
```
Comment (mmoll) [inline dependencies/trusty/bundler/bundler.dirs]: this was the only change additional to the changelog

**Comment by domcleal** [comment]
[test]

**Comment by domcleal** [comment]
[test]

**Comment by domcleal** [comment]
Merged and [building](http://ci.theforeman.org/job/packaging_build_deb_dependency/1424/), thanks @mmoll.

**Pull request merged**